### DISCUSSION TITLE
Install Extension on Postgres component

### DISCUSSION THREAD

**Discussion by vicperdana** [microsoft/aspire#4002] [Q&A] [answered]
Has anyone tried to install Postgres extension via Aspire?  

I'd like to run this command against the Postgres docker image as part of docker build.
`CREATE EXTENSION vector;`

Is this something supported via `azd`?

Thanks!

**Comment by vicperdana** [comment, marked as answer]
Can solve this by choosing a different image that has the required extension with `.WithImage()`

**Comment by davidfowl** [comment]
Related https://github.com/dotnet/aspire/issues/3052
